### PR TITLE
Evrard glass sphere

### PR DESCRIPTION
**Pull request by sekelle**

### CODE REVIEW

**Review by jgphpc — APPROVED** [review]

**Review by jaescartin1 — APPROVED** [review]

File: main/src/init/noh_init.hpp [hunk 1]
```diff
@@ -42,15 +42,12 @@ namespace sphexa
...
 {
     using T = typename Dataset::RealType;
 
-    int    ng0         = 100;
-    double r1          = constants.at("r1");
-    double totalVolume = std::pow(2 * r1, 3);
-    double hInit       = std::cbrt(3.0 / (4 * M_PI) * ng0 * totalVolume / d.numParticlesGlobal) * 0.5;
-
+    int    ng0           = 100;
+    double hInit         = std::cbrt(3.0 / (4 * M_PI) * ng0 * totalVolume / d.numParticlesGlobal) * 0.5;
```
Comment (rmcabezon) [inline main/src/init/noh_init.hpp:50]: This formula is assuming constant density. It is more general to use local values: i.e. hInit = std::cbrt(3 / 4 / PI * ng0 * m[i] / rho[i]) * 0 .5

Comment (rmcabezon) [reply main/src/init/noh_init.hpp:50]: Which for Noh is true, but better to keep it general.

**Review by rmcabezon — COMMENTED** [review]
I suggested a change in the h calculation

File: main/src/init/noh_init.hpp (hunk 1 above)
Comment (sekelle) [reply main/src/init/noh_init.hpp:50]: For one, this is not general code but Noh-specific initialization.
Two, we don't have rho[i] at initialization time.

**Pull request merged**